param/parameterized.py

```diff
@@ -963,6 +963,10 @@ def _sorter(p):
         cls.__signature__ = new_sig
 
 
+def is_mutable(value):
```

Comment:
Param has now a `_utils.py` module in which this utility could live. Also I've been trying to make any new function/class private not to pollute the namespace. So including your suggestion I'd rename it `_is_mutable_container` and move it to `_utils.py`.

Reply:
Sounds good, thanks!